src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -238,6 +240,20 @@ public void updateFilteredPilotList(Predicate<Pilot> predicate) {
 
     //=========== Location ========================================================
 
+    @Override
+    public Location getLocationById(String id) {
+        Location locationToFind = null;
```

Comment:
Let's not use `null` here and instead use Optional. `null` is not safe. I think this issue is shared also in the pr #77. Since that PR has already been merged, let's just keep it this way and we can fix this later.